src/css/base/_variables_coa.scss

```diff
@@ -32,7 +32,8 @@ $coa-textwrap-width: 545px;
 $coa-color-blue: #005EFF;
 // PURPLES
 $coa-color-light-purple: #D7D2D9;
-$coa-color-purple: #9B90A2;
+$coa-color-another-grayish-purple: #EAEAEA;
```

Comment:
Can you give my branch a look for colors -- I've formalized colors based on Celine's stickersheet. https://github.com/cityofaustin/janis/blob/fb35ba5e62f834729f833345dfb06d113559e5d1/src/css/base/_base.scss